specification/network/resource-manager/Microsoft.Network/stable/2024-07-01/expressRouteCircuit.json

```diff
@@ -1678,6 +1678,45 @@
       },
       "description": "Response for ListAuthorizations API service call retrieves all authorizations that belongs to an ExpressRouteCircuit."
     },
+    "AdvertisedPublicPrefixProperties": {
+      "type": "object",
+      "properties": {
+        "prefix": {
+          "type": "string",
+          "description": "Advertised Public Prefix."
+        },
+        "validationId": {
+          "type": "string",
+          "description": "Validation string that contains circuit peering info."
+        },
+        "signature": {
```

Comment:
>signature [](http://example.com/codeflow?start=9&length=9)

This implies that validationId is a secret. If so, it (the validationId property) needs to be marked with x-ms-secret and its value must be nulled/blanked in GET payloads. If it needs to be retrieved, that must be done via a separate list* POST action instead of GET.

Reply:
validationId is not a secret. It is just a plain text that contains information that is already available to the customers. This is just a well formatted string that the customer can use in the validation process.